README.md

```diff
@@ -78,25 +54,56 @@ Do this once:
 mv src/third_party/libreofficekit{,.old}
 ```
 
-Linux/MacOs
+**Linux/macOS**
 ```bash
 # link src/third_party/libreofficekit to your local libreofficekit/libreoffice-core
 ln -s ../libreofficekit/libreoffice-core src/third_party/libreofficekit
 ```
 
-Windows
+**Windows**
 ```bash
 # note you must use an administrator command prompt, and the source and target are flipped
 mklink /D ".\src\third_party\libreofficekit" "..\libreofficekit\libreoffice-core"
 ```
 
+When you make a new local build of `libreofficekit`, make a new Electron build with `UPDATE_LOK=true`:
+```shell
+UPDATE_LOK=true scripts/e build
+```
+
+## Using a debug build in another project
+```shell
+# copy zipped runtime files to ~/another_project
+INJECT_TO="~/another_project" scripts/e build
+
+# this copies a fresh build of dist.zip, mksnapshot.zip, mksnapshot_args, and gen/v8/electron.S
+# dist.zip can be expanded into node_modules/electron/dist
+# mksnapshot.zip can be expanded into node_modules/electron-mksnapshot/bin
+# all other files can be copied into node_modules/electron-mksnapshot/bin
+```
+
+## Debugging
 
-When you make a new local build of `libreofficekit`, remove the old build in electron-libreoffice's output:
 ```shell
-rm -rf src/out/Default/libreofficekit
+# Run the QA testing ground
+scripts/e run --renderer-startup-dialog qa/index.html
+
+# On Linux/Mac, it will report the PID of the renderer,
+# then you can start GDB with that PID. Assuming the renderer's PID is 1163:
+gdb -p 1163 -ex 'signal SIGUSR1'
```

Comment:
do you think it's worth adding a LLDB command for Mac since that is the default debugger?

Reply:
Getting Xcode setup is probably better on macOS. That's a whole other can of worms than this PR, probably needs its own file honestly.

Reply:
Yeah now that you mention it some Xcode documentation would be great